### ISSUE TITLE
Provide compatibility with DF 0.34.01

### ISSUE THREAD

**Issue by no-identd** [TroZ/DF2MC#2]
YOU KNOW YOU WANT IT BABY.

**Comment by windlessuser** [comment]
yeah, this would be really cool, and with the newer version of minecraft as well

**Comment by AceofCrafters** [comment]
The new minecraft compatability isnt an issue just downgrade minecraft to beta 1.4 with mcnostalgia, convert the world with that version of minecraft, then update minecraft and convert the world again. Although the compatability with the new DF version will be amazing. It says he is working on it in the wiki.
